templates/model/manifest/v1alpha1/manifest.go

```diff
@@ -87,7 +100,7 @@ func (i *Input) Validate() error {
 // OutputHash records a checksum of a single file as it was created during
 // template rendering.
 type OutputHash struct {
-	Pos model.ConfigPos
+	Pos model.ConfigPos `yaml:"-"`
```

Comment:
Nits: comment?

Reply:
This is a common pattern for all the model structs, I don't think it needs to be documented every time.